### PR TITLE
Home screen / Landing Page

### LINKED ISSUE #2: landing screen on home page
image

name is developer station

### PR DESCRIPTION
**Pull request by hackinrish**
Screenshot 2024-10-21 at 3 44 27 PM

- Created landing screen for home page based on the figma with minor positional differences in image elements
- Updated image above

- Closes issue #2

### CODE REVIEW

**Comment by shahdivyank** [comment]
merge conflicts

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,7 +1,67 @@
...
+            ></Image>
+            <h1 className="w-full text-5xl text-yellow-300">
+              DEVELOPER STATION
+            </h1>
+          </div>
+        </div>
+      </div>
+
+      <div className="relative h-1/2 bg-blue-950">
+        <Image
```
Comment (shahdivyank) [inline src/app/page.tsx]: self closing image

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,7 +1,67 @@
...
-      Hello World
+    <div className="flex h-screen w-screen flex-col bg-blue-950">
+      <div className="pb-4">
+        <div className="h-1/8 flex flex-row justify-between p-4">
+          <div className="h-8 w-8 rounded-full bg-gray-300"></div>
+          <div className="ml-32 h-8 w-24 bg-gray-300 text-center">
+            <p className="bg-gray-300 p-2">LOGO</p>
+          </div>
+          <div className="flex flex-row space-x-2">
+            <div className="h-8 w-8 rounded-full bg-gray-300"></div>
```
Comment (shahdivyank) [inline src/app/page.tsx:19]: self closing

File: public/landingpagetrainpath.png
Comment (shahdivyank) [inline public/landingpagetrainpath.png:1]: making a landing folder for the landing images

**Comment by stanleylew5** [comment]
image

make sure you add a gradient instead of using blue-950, feel free to add the colors within tailwind config file. and also increase the size of the "developer station" text

**Review by stanleylew5 — CHANGES_REQUESTED** [review]
change the gradient to go from top to bottom since that's what the figma shows, also you're missing an element 
image
that circle thingie. also merge the pr that'll convert your png to webp that's related to this pr

**Pull request merged**